### DISCUSSION TITLE
AospExtended-12.x sync error

### DISCUSSION THREAD

**Discussion by romgharti** [ROM-builders/temporary#8946] [Q&A]
https://cirrus-ci.com/build/5642466069053440

**Comment by Apon77** [comment]
> https://cirrus-ci.com/build/5642466069053440

try to sync locally first.

**Comment by romgharti** [comment]
![image_2022-04-23_13-59-54](https://user-images.githubusercontent.com/21030085/164886456-a6687276-028e-4322-b8e1-304b6f700ee0.png)
